File: 260111/배수의 개수/number-of-multipliers.js
const fs = require('fs');
const input = fs.readFileSync(0).toString().trim().split('\n').map(Number);

let c3 = 0;
let c5 = 0;

for (let i = 0; i < 10; i++) {
    if (input[i] % 3 === 0 && input[i] % 5 === 0){
        c3++;
        c5++;
    } else if (input[i] % 3 === 0) {
        c3++;
    } else if (input[i] % 5 === 0) {
        c5++;
    }
}

console.log(c3, c5);

File: 260111/배수의 개수/number-of-multipliers.py
c3 = 0
c5 = 0

for _ in range(10):
    num = int(input())
    if num % 3 == 0 and num % 5 == 0:
        c3 += 1
        c5 += 1
    elif num % 3 == 0:
        c3 += 1
    elif num % 5 == 0:
        c5 += 1

print(c3, c5)